.travis.mk

```diff
@@ -374,10 +380,17 @@ test_osx_no_deps: build_osx
 	${INIT_TEST_ENV_OSX}; \
 	cd test && ./test-run.py --vardir ${VARDIR} --force $(TEST_RUN_EXTRA_PARAMS)
 
+test_osx_arm64_only:
+	make PUC-Rio-Lua-5.1-tests
+
 test_osx: deps_osx test_osx_no_deps
 
 test_osx_github_actions: deps_osx_github_actions test_osx_no_deps
 
+test_osx_arm64_github_actions: deps_osx_github_actions build_osx test_osx_arm64_only
+
+test_osx_debug_arm64_github_actions: deps_osx_github_actions build_osx_debug test_osx_arm64_only
```

Comment:
After the changes related to `build_osx_debug`, the introduced parameters need to be set in these rules. The change is also not complex (you can also find them on [my branch](https://github.com/tarantool/tarantool/commit/4cbdb5a)):
```diff
@@ -381,6 +381,14 @@ test_osx_no_deps: build_osx

 test_osx_github_actions: deps_osx_github_actions test_osx_no_deps

+test_osx_arm64_github_actions: BUILD_TYPE=RelWithDebInfo
+test_osx_arm64_github_actions: BUILD_TYPE_FLAGS=-DENABLE_WERROR=ON
+test_osx_arm64_github_actions: deps_osx_github_actions test_osx_arm64_no_deps
+
+test_osx_debug_arm64_github_actions: BUILD_TYPE=Debug
+test_osx_debug_arm64_github_actions: BUILD_TYPE_FLAGS=
+test_osx_debug_arm64_github_actions: deps_osx_github_actions test_osx_arm64_no_deps
+
 # Static macOS build

 STATIC_OSX_PKGS=cmake
```

Reply:
Done.